hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java

```diff
@@ -1091,6 +1092,36 @@ public void deleteObjectTagging(String keyName) throws IOException {
     proxy.deleteObjectTagging(volumeName, name, keyName);
   }
 
+  /**
+   * Gets the lifecycle configuration information.
+   * @return OmLifecycleConfiguration or exception is thrown.
+   * @throws IOException
+   */
+  @JsonIgnore
+  public OzoneLifecycleConfiguration getLifecycleConfiguration()
+      throws IOException {
+    return proxy.getLifecycleConfiguration(volumeName, name);
+  }
+
+  /**
+   * Creates a new lifecycle configuration.
```

Comment:
Can we add a comment here about "It will overwrite an existing lifecycle configuration?